tests/sentry/incidents/test_subscription_processor.py

```diff
@@ -600,6 +605,182 @@ def test_seer_call(self, mock_seer_request: MagicMock):
             ],
         )
 
+    @mock.patch(
+        "sentry.seer.anomaly_detection.get_anomaly_data.SEER_ANOMALY_DETECTION_CONNECTION_POOL.urlopen"
+    )
+    @with_feature("organizations:incidents")
+    @with_feature("organizations:anomaly-detection-alerts")
+    @with_feature("organizations:anomaly-detection-rollout")
+    @with_feature("organizations:workflow-engine-metric-alert-processing")
+    def test_seer_call_dual_processing(self, mock_seer_request: MagicMock):
```

Comment:
can we decompose this test into multiple bits? it's a bit hard for me to tell how this is testing seer calls -- maybe we could pull stuff out into setup methods and `verify_deserialized_body` into a method to DRY it a little?

Reply:
I broke this out into 3 tests but the next PR I want to work on is breaking this gigantic test file out into like 4 files and refactoring the workflow engine parts https://getsentry.atlassian.net/browse/ACI-392 so I'm gonna punt cleaning it up in this one.